File: NMlib/TestTrainer64/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using NMlib64;

namespace TestTrainer64
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void Timer1_Tick(object sender, EventArgs e)
        {
            NeutronMemoryLibrary.IslemEkle("İŞLEM İSMİ");
            if (NeutronMemoryLibrary.ProcessRunning)
            {
                label1.Text = "İşlem çalışıyor.";
            }
            else
            {
                label1.Text = "İşlem çalışmıyor.";
            }
        }

        private void CheckBox1_CheckedChanged(object sender, EventArgs e)
        {
            NeutronMemoryLibrary.OpcodeKullan(0x18B59E, "90 90 90 90");
        }

        private void TrackBar1_Scroll(object sender, EventArgs e)
        {
            NeutronMemoryLibrary.PointerKullan(0x3B4FA0, 0xB10, 0x180, 0x219, 1);
        }
    }
}

File: NMlib/TestTreainer64VB/Form1.vb
Imports NMlibVB
Public Class Form1
    Private Sub Timer1_Tick(sender As Object, e As EventArgs) Handles Timer1.Tick

        NMlib64VB.NeutronMemoryLibrary.IslemEkle("İŞLEM İSMİ")
        If NMlib64VB.NeutronMemoryLibrary.ProcessRunning = True Then
            label1.Text = "İşlem çalışıyor."
        Else
            label1.Text = "İşlem çalışmıyor."
        End If
    End Sub

    Private Sub CheckBox1_CheckedChanged(sender As Object, e As EventArgs) Handles checkBox1.CheckedChanged
        NMlib64VB.NeutronMemoryLibrary.OpcodeKullan(&H18B59E, "90 90 90 90")
    End Sub

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        Timer1.Start()
    End Sub

    Private Sub TrackBar1_Scroll_1(sender As Object, e As EventArgs) Handles TrackBar1.Scroll
        NMlib64VB.NeutronMemoryLibrary.PointerKullan(&H3B4FA0, &HB10, &H180, &H219, 1)
    End Sub
End Class
